README.md

```diff
@@ -183,6 +183,7 @@ gplot(h)
 + `arrowangleoffset` Angular width in radians for the arrows. Default: `π/9 (20 degrees)`
 + `linetype` Type of line used for edges ("straight", "curve"). Default: "straight"
 + `outangle` Angular width in radians for the edges (only used if `linetype = "curve`). Default: `π/5 (36 degrees)`
++ `backgroundc` Color for the plot background. Default: `nothing`
```

Comment:
I know that for other params we alse use the `c` postfix, but maybe we should just call it `background_color`.

Reply:
yes! Should we think about changing the kwargs to being more verbose? `nodefill_color`, etc.